src/api/nft-api.ts

```diff
@@ -264,7 +266,7 @@ export async function getNftsForCollection(
 ): Promise<CollectionNftsResponse | CollectionBaseNftsResponse> {
   const withMetadata = omitMetadataToWithMetadata(options?.omitMetadata);
   const response = await requestHttpWithBackoff<
-    GetNftsForCollectionAlchemyParams,
+    GetOwnersForCollectionAlchemyParams,
```

Comment:
Why is this within `getNftsForCollection`? Think you want a new function `getOwnersForCollection`.

Reply:
oops, forgot to rename back. Thanks for catching!